Feature/api6

zealotnt opened the pull request.

zealotnt committed:
implement API shipmentVehicleReport

zealotnt committed:
add blockchain-clear script

zealotnt committed:
add script to debug hyperledger chaincode

zealotnt committed:
retouch warehouseReport API

zealotnt commented on blockchain/models/org.bitship.cto:
```diff
@@ -100,8 +100,7 @@ transaction ShipmentVehicleReport {
 transaction WarehouseReport {
   --> Inspector inspector
   --> Package[] packages
-  --> Warehouse warehouse
```
wareHouse có thể get từ inspector, vậy có truyền thêm wareHouse có vẻ thừa ?

tiengtinh replied:
yes, rice

Thanghand commented on blockchain/lib/logic.js:
```diff
@@ -61,6 +63,66 @@ async function shipmentTransfer(shipmentTransfer) {
+        } else {
+            packageObject.status = "LOST";
+            event.message = "Lost";
+        }
```
We should compare list of packages to ShipmentVehicle.packages. If package does not contain in ShipmentVehicle.packages  => Change status to LOST

zealotnt committed:
modify shipmentVehicleReport API

+ so that inspector only send once, and inspector cannot send status
+ the missing packages from the API request is consider to be LOST
